### DISCUSSION TITLE
Add custom labels for emails/telephones

### DISCUSSION THREAD

**Discussion by luison** [twentyhq/twenty#9088] [Ideas]
As requested in this Discord thread... https://discord.com/channels/1130383047699738754/1130383048173682821/1308133745848487936 this is a request to add labels to the different emails for persons/companies.

This is one of our limitations today, to migrate to twenty.

## Current behavior
Recently added multiple emails but not labelling system to identifying them.
Google uses this for all contact data... including telephones, etc

**Example:** 
 We use this on companies to identify departments so we can use our CRM as a central contact BBDD for our systems.  We identify things like... "Bookings", "Accounts", etc. In this way when sending an email depending on the context we can request via API the updated email/emails.

![Image](https://github.com/user-attachments/assets/d2f9309b-f955-4c1e-b01d-69eba039c7cb)

## Expected behavior
Each email has a "label" associated to it. Labels should either be free text or based on a preference system, but ideally allowing an "other" that could be personalised. This could actually be abstracted to phone, numbers, addresses, etc that support more than one value.

Label values should be included wherever the email is referenced, including API, webhooks, etc

**Comment by FelixMalfait** [comment]
Thanks! cc @Bonapara

**Comment by Bonapara** [comment]
Thanks, @luison! Great suggestion. We won't implement it in the short term, so I'm starting a discussion to prioritize it if there's more demand
